sponsors.html

```diff
@@ -15,4 +15,16 @@ <h3 class="clemson-orange-text">By sponsoring, you gain ...</h3>
   </ul>
   <h3 class="clemson-orange-text">How to sponsor</h3>
   <p class="flow-text">For more information on ways to become involved with <strong>CUCTF 2020</strong>, please download our <a href="/public/files/CUCTF_Sponsorship_Packet.pdf" download="">Sponsor Packet</a> or contact us at <a href="mailto:sponsor@cuctf.io">sponsor@cuctf.io</a>.</p>
+  <br>
+  <h4 class="clemson-orange-text">CUCTF 2020 Contingency Plan for Sponsors</h4>
```

Comment:
Make this consistent with the other headings in the page. Randomly having a centered h4 heading unrelated to the h3 above looks super weird.

```suggestion
  <h3 class="clemson-orange-text">CUCTF 2020 Contingency Plan for Sponsors</h3>
```